README.md

```diff
@@ -12,6 +12,23 @@ Main motivation for Intl.DisplayNames project was to enable developers to get tr
 * Reduce translation cost for developers.
 * Consistent translation of language, region and script display name on the web.
 
+### Scope
+This proposal is not intended to provide a translation for all strings, but rather focus on particular items which are application-independent. There are two classes of strings we're considering here:
+
+- Strings that are already available because they're needed for other APIs. For example, names for the days of the week are necessary to provide Intl.DateTimeFormat capabilities. In the absence of a direct API, we see programs parsing the output of Intl.DateTimeFormat to find these names, which is an unreliable/unstable technique.
```

Comment:
I'd prefer to see "parsing the output of Intl.DateTimeFormat" labelled more clearly as an anti-pattern.

Reply:
While I agree with you that is a good idea, I think such statement about such parsing is an "anti-pattern" itself is "out of the scope" to the introduction of this particular API. I suggest you make a PR to ECMA402 itself, in particular in the introduction section of Intl.DateTimeFormat and other formatters.